specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1249,284 +2194,405 @@
         "ice": {
           "$ref": "#/definitions/WeatherUnit",
           "description": "Ice"
-        },
-        "cloudCover": {
-          "type": "integer",
-          "format": "int32",
-          "description": "Percent representing cloud cover."
         }
       }
     },
-    "HourlyForecastResponse": {
+    "QuarterDayForecastResult": {
       "type": "object",
       "properties": {
         "forecasts": {
+          "readOnly": true,
           "type": "array",
-          "description": "Forecast data for each returned hour.",
+          "description": "Forecast data for each quarter in the response.",
+          "items": {
+            "$ref": "#/definitions/QuarterDayForecast"
+          }
+        }
+      }
+    },
+    "CurrentConditionsResult": {
+      "type": "object",
+      "properties": {
+        "results": {
```

Comment:
Copied over from APIView:

> **tjprescott**
> You should be specific because the non-tropic storm APIs are being specific.
> 
> **john35452**
> We have a discussion internally, and we would like to use results for all these three new weather apis.

This is not an arbitrary choice you can make just for new APIs. The names need to be consistent with the entire maps effort, which has already decided in the direction of specific collections over "results".

Reply:
Per our discussion, it's fine to use `x-ms-client-name` to address this. No service change is needed.

Reply:
Sure. I have applied meaningful name in x-ms-client-name in the below commit.
https://github.com/Azure/azure-rest-api-specs/pull/16802/commits/92e55e145299c94d8b87c9e3bf319d3feba6f386